starter-code/public/scripts/controllers/articleController.js

```diff
@@ -5,8 +5,12 @@ var app = app || {};
   const articleController = {};
   // TODO: Setup a function that kicks off the fetching and rendering of articles, using the same
   // code that used to be in index.html.
-  // Also be sure to hide all the main section elements, and reveal the #articles section:
-
+  // Also be sure to hide all the main section elements, and reveal the #articles section: DONE
+  articleController.index = () => {
+    app.Article.fetchAll(app.articleViewInitIndexPage);
+    $('main section').first().hide();
```

Comment:
Here you are only hiding the first element in main section, which is article. Then you are showing it again. So your about section still shows up on your page.